conftest.py

```diff
@@ -0,0 +1,5 @@
+import sys
```

Comment:
I suppose github action do not read conftest as CI is using unittest to run tests :-/ (introduce by this commit https://github.com/camptocamp/pytest-odoo/commit/ea535606bc3a1b6d964cabcf479d1387c66d694f#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR42)

It seems to be possible to test pytest plugin using pytest, there is also a pytester pluging, not sure we needs it https://docs.pytest.org/en/stable/how-to/writing_plugins.html#testing-plugins so I suppose we needs to change how github action launch tests on CI as well.

Reply:
The issue is that we were using unittest in the ci.
I think it makes more sense to use pytest to use a pytest mod.

Reply:
After few testing, I replaced the mock using testconf.py by a setup with tox and a mock module.
This seems to play nicely.

Also had to make few changes to fixup the coverage and codcov config.